extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonJsonTransformerImpl.java

```diff
@@ -0,0 +1,26 @@
+package org.dtls.fairifier;
+
+
+public class RdfSkeletonJsonTransformerImpl implements RdfSkeletonTransformer{
```

Comment:
As I've said with the setup of the interface, I'm not sure whether this implementation is according to our overall design.